mapie/tests/test_classification.py

```diff
@@ -1370,3 +1370,56 @@ def test_error_raps_cv_not_prefit(cv: Union[int, None]):
     mapie = MapieClassifier(method="raps", cv=5)
     with pytest.raises(ValueError, match=r".*RAPS method can only.*"):
         mapie.fit(X_toy, y_toy)
+
+
+def test_not_all_label_in_calib():
+    """Test that the true label cumsumed probabilities
```

Comment:
I like it better like this:
"""
    Test
"""

Reply:
done